components/sheet/src/sheet.tsx

```diff
@@ -28,7 +28,7 @@ const SheetOverlay = forwardRef<
 SheetOverlay.displayName = SheetPrimitive.Overlay.displayName;
 
 const SheetVariants = cva(
-	"fixed z-50 flex flex-col bg-background shadow-lg transition ease-in-out data-state-open:animate-in data-state-closed:animate-out data-state-closed:duration-100 data-state-open:duration-100",
+	"fixed z-50 flex flex-col bg-base shadow-lg transition ease-in-out data-state-open:animate-in data-state-closed:animate-out data-state-closed:duration-100 data-state-open:duration-100",
```

Comment:
This elevation is feeling a little dingey in this context. It's not actually a background, it's more of a card. I think it ought to be `white` in light and `050` in dark. Or something. `bg-modal` or something? Or maybe just `bg-card`.

Reply:
Yeah, added a `bg-dialog` for these and it is exactly that now